roboslack-webhook/src/main/java/com/palantir/roboslack/webhook/SlackWebHookService.java

```diff
@@ -69,31 +69,33 @@ public static SlackWebHookService with(WebHookToken token, String userAgent) {
         return new SlackWebHookService(token, userAgent);
     }
 
+    private Call<ResponseCode> sendCall(MessageRequest messageRequest) {
+        return webHook.sendMessage(token.partT(), token.partB(), token.partX(), messageRequest);
+    }
+
     /**
-     * We can't serialize the response correctly since Slack only sends back a 'text/html' string,
-     * so we manually pull it from the {@link ResponseBody} instead.
+     * Sends a message to a connected {@link SlackWebHookService} asynchronously using provided {@link Callback}.
+     *
+     * @param messageRequest the {@link MessageRequest} to execute sending
+     * @param callback the {@link Callback} to trigger on response
      */
-    private static String executeCallAndGetResponseBody(Call<ResponseBody> call) {
-        try {
-            ResponseBody body = call.execute().body();
-            if (body != null) {
-                return body.string();
-            }
-            return "";
-        } catch (IOException e) {
-            throw new RuntimeException("Unable to execute call", e);
-        }
+    public void sendMessage(MessageRequest messageRequest, Callback<ResponseCode> callback) {
```

Comment:
nit: would it be worth calling this `sendMessageAsync` to make the difference more explicit?